### DISCUSSION TITLE
Failed to follow procedure from "Getting Started"

### DISCUSSION THREAD

**Discussion by zhil** [calimero-network/core#804] [Starknet Hackathon]
I am trying to login into admin area based on https://calimero-network.github.io/getting-started/admin-dashboard

I created new MetaMask account, but I cant use it and there is no visible error.
![image](https://github.com/user-attachments/assets/223ec5d3-1e2f-4d50-ae92-6a3824b186ae)

On screenshot you have etherium too with 0 balance, but I dont have "Sign transaction button" 

I have recorded gif

![calimero](https://github.com/user-attachments/assets/32f83ebd-f418-45c9-aae6-19ed5f6bc4be)

**Comment by frdomovic** [comment]
Hej @zhil, 
The issue here is that the web application failed to fetch login request challenge. You can see that in console and network.
The only way I managed to recreate this was reseting local storage in browser console with "localStorage.clear()" which ended up reseting the default values login process needs.
To fix this you can simply go back to root path "/admin-dashboard/" and setup the node URL again.
Here are images and video for reference

https://github.com/user-attachments/assets/ce314271-e5df-45a4-bfa6-2f4317b6e787

<img width="1510" alt="Screenshot 2024-10-16 at 10 14 38" src="https://github.com/user-attachments/assets/443199f0-dfaa-4f92-b7ed-7b3a2cba0d3a">
<img width="1511" alt="Screenshot 2024-10-16 at 10 15 03" src="https://github.com/user-attachments/assets/a76f31f7-9e30-4e25-998b-22ca4316a267">

**Comment by MatejVukosav** [comment]
Hi @zhil 

Were you able to fix the issue?

**Reply by zhil** [reply]
I didn't test yet

**Comment by MatejVukosav** [comment]
Hackathon has finished
